File: src/test/java/dev/kikugie/testmod/ModInit.java
package dev.kikugie.testmod;

import dev.kikugie.testmod.command.RenderCommand;
import net.fabricmc.api.ModInitializer;
import net.fabricmc.fabric.api.command.v2.CommandRegistrationCallback;

public class ModInit implements ModInitializer {
    @Override
    public void onInitialize() {
        CommandRegistrationCallback.EVENT.register(((dispatcher, registryAccess, environment) -> RenderCommand.register(dispatcher)));
    }
}

File: src/test/kotlin/dev/kikugie/testmod/ModInit.kt
package dev.kikugie.testmod

import net.fabricmc.api.ModInitializer
import net.fabricmc.fabric.api.command.v2.CommandRegistrationCallback

object ModInit : ModInitializer {
    override fun onInitialize() {
        CommandRegistrationCallback.EVENT.register(RenderCommand::register)
    }
}